src/main/java/echoServer/Echoer.java

```diff
@@ -1,5 +1,8 @@
 package echoServer;
 
+import echoServer.HTTP.HTTPRequest;
+import echoServer.HTTP.RequestParser;
+import echoServer.HTTP.Router;
```

Comment:
At some point, you're going to want to rename everything (packages, folders, etc.) to `httpServer`! Echo server evolved!

Reply:
Renamed everything!